use-cases/vehicle-routing/src/test/java/org/acme/vehiclerouting/domain/geo/HaversineDrivingTimeCalculatorTest.java

```diff
@@ -0,0 +1,25 @@
+package org.acme.vehiclerouting.domain.geo;
+
+import org.acme.vehiclerouting.domain.Location;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class HaversineDrivingTimeCalculatorTest {
+
+    private final DrivingTimeCalculator drivingTimeCalculator = new HaversineDrivingTimeCalculator();
+
+    // Results have been verified with the help of https://latlongdata.com/.
+    @Test
+    void calculateDrivingTime() {
+        Location Gent = new Location(51.0441461, 3.7336349);
+        Location Brno = new Location(49.1913945, 16.6122723);
```

Comment:
Lovely!

Reply:
It was already there, so I cannot take credit for it, but I love it as well :)